integreat_cms/cms/templates/feedback/_feedback_widget.html

```diff
@@ -25,22 +25,31 @@
                     {% if feedback.object_url %}
                         <a href="{{ feedback.object_url }}">{{ feedback.object_name }}</a>
                     {% else %}
-                        {{ feedback.object_name }}
+                        <div class="feedback-entry">
+                            <span class="feedback-entry-content overflow-hidden overflow-ellipsis whitespace-nowrap">
+                                {{ feedback.object_name }}
+                            </span>
+                            <a class="btn-icon inline"
+                               title="{% trans "Copy to clipboard" %}"
+                               data-copy-to-clipboard="{{ feedback.object_name }}">
+                                <i icon-name="copy"></i>
+                            </a>
+                            <a class="toggle-feedback-entry">
+                                <i icon-name="chevron-down"></i>
+                            </a>
+                        </div>
                     {% endif %}
                 </td>
-                <td class="pr-2 feedback-comment max-w-[75px] lg:max-w-[300px] xl:max-w-[75px] 3xl:max-w-[300px] 4xl:max-w-[500px]"
+                <td class="pr-2 feedback-entry max-w-[75px] lg:max-w-[300px] xl:max-w-[75px] 3xl:max-w-[300px] 4xl:max-w-[500px]"
```

Comment:
```suggestion
                <td class="pr-2 max-w-[75px] lg:max-w-[300px] xl:max-w-[75px] 3xl:max-w-[300px] 4xl:max-w-[500px]"
```
Similar problem to the admin feedback list last time - I think setting this class on the table row leads to the feedback comment to overflow into the following columns.

Reply:
I was so sure that I removed it everywhere. Thx. For this thorough review :D